src/components/Lobby/Room/Room.jsx

```diff
@@ -10,15 +10,15 @@ export class Room extends React.Component {
         const waitingVisible = !this.props.room.started && (this.props.room.minPlayers - this.props.room.players.length > 0);
         return (
             <div className={styles.room} key={this.props.room.roomId} data-cy="room">
-                <h2 data-cy="room-title">{"Room: " + this.props.room.roomId}</h2>
-                <ul id="players" data-cy='room-players'>
-                    {this.props.room.players.map((player, key) => <li key={key}><span data-cy='player-username'>{player.username}</span></li>)}
+                <h2 className={styles.roomHeader} data-cy="room-title" >{"Room: " + this.props.room.roomId}</h2>
```

Comment:
You've added a random space after room title

Reply:
oopsie woopsie